enterprise/admin/views.py

```diff
@@ -0,0 +1,119 @@
+"""
+Custom Django Admin views used in enterprise app.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import json
+
+from django.contrib import admin
+from django.contrib.auth import get_permission_codename
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
+from django.views.generic import View
+
+from enterprise.admin.forms import ManageLearnersForm
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser
+
+
+class EnterpriseCustomerManageLearnersView(View):
+    """
+    Manage Learners view.
+
+    Lists learners linked to chosen Enterprise Customer and allows adding and deleting them.
```

Comment:
@zubair-arbi There's no way to enforce it (pydocstyle does not support that yet), so I would suggest we drop that suggestion for now.